ui/app/src/services/sites.ts

```diff
@@ -34,7 +34,7 @@ export function fetchSites(paginationOptions?: PaginationOptions): Observable<Pa
     map(({ response }) => Object.assign(
       response.sites.map(site => ({
         id: site.siteId,
-        name: site.siteId,
+        name: site.siteId,    // TODO: need to get site.name when API is ready
```

Comment:
It's not so we can leave this this way until backend is ready.

@dejan-brkic FYI ☝️ this is the place to change in case you want to take not and test/change when you get to this.

Reply:
site.name ?? site.siteId